assets/js-examples-new/examples/with-page-objects/google.js

```diff
@@ -27,6 +27,6 @@ describe('google search with consent form - page objects', function() {
     resultsPage.expect.section('@menu').to.be.visible;
 
     const menuSection = resultsPage.section.menu;
-    menuSection.expect.element('@all').to.be.visible;
+    menuSection.expect.element('@maps').to.be.visible;
```

Comment:
Maybe replace `@maps` with `@videos`? That makes more sense in the context of 'Nightwatch.js' being googled.